src/lib/phdi-building-blocks/phdi_building_blocks/conversion.py

```diff
@@ -275,9 +282,25 @@ def convert_message_to_fhir(
             {"name": "rootTemplate", "valueString": root_template},
         ],
     }
-    response = requests.post(
-        url=url, json=data, headers={"Authorization": f"Bearer {access_token}"}
+    access_token = cred_manager.get_access_token().token
```

Comment:
It seems like this section of code is repeated in other files. Perhaps it should be moved out and imported